tsconfig.json

```diff
@@ -14,10 +14,11 @@
     "preserveConstEnums": true,
     "sourceMap": true,
     "strictNullChecks": true,
```

Comment:
This change means you can rm lines 10 and 16

Reply:
I removed the redundant `strictNullChecks` and `noImplicitAny` options from `tsconfig.json`